client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -187,14 +169,6 @@ registerHandlers( 'state/data-layer/wpcom/sites/media/index.js', {
 		} ),
 	],
 
-	[ MEDIA_ITEM_UPDATE ]: [
-		dispatchRequest( {
-			fetch: updateMedia,
-			onSuccess: updateMediaSuccess,
-			onError: updateMediaError,
```

Comment:
The `updateMedia` used to create and pass an `originalMediaItem` object and then `updateMediaError` reverted the state back to the original item. I.e., optimistic updates. But I now verified that the `updateMedia` workflow doesn't really do any optimistic updates. Only the `editMedia` workflow does, and it shares the `updateMedia*` handlers with `updateMedia`. :+1: